src/Raven.Server/Routing/RouteInformation.cs

```diff
@@ -106,6 +107,21 @@ public Task CreateDatabase(RequestHandlerContext context)
             await Task.WhenAny(database, Task.Delay(time));
             if (database.IsCompleted == false)
             {
+                if (databasesLandlord.InitLog.TryGetValue(databaseName, out var initLogQueue))
+                {
+                    // lets first check if database is in recovery process. if so - lets throw appropriate exception
+                    if (initLogQueue.TryDequeue(out var str))
```

Comment:
This is destructive process.
We want to keep getting the same thing over and over.